packages/react-core/src/components/DualListSelector/DualListSelectorPane.tsx

```diff
@@ -65,6 +65,8 @@ export interface DualListSelectorPaneProps {
   searchInputAriaLabel?: string;
   /** @hide Callback for updating the filtered options in DualListSelector. To be used when isSearchable is true. */
   onFilterUpdate?: (newFilteredOptions: React.ReactNode[], paneType: string, isSearchReset: boolean) => void;
+  /** Height of the list of options rendered in the pane. **/
+  listHeight?: string;
```

Comment:
Would something along the lines of `listMinHeight`/`listMinimumHeight` be more fitting, since it's changing the minHeight in CSS?